recipe/meta.yaml

```diff
@@ -57,4 +55,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - rolando
+    - rmax
```

Comment:
Should send similar PRs to anything else you are maintaining that hasn't already been updated. Feel free to add `[ci skip]` in the commit messages of such PRs if only the user name is changing.